### ISSUE TITLE
How to backup docker volumes with Kopia?

### ISSUE THREAD

**Issue by akash07k** [kopia/kopia#2142]
I want to use Kopia because it is the best application to backup the stuff in my opinion.
I'm managing my own server, and I need to backup the data of my docker container volumes.
Is there any way by which we can take the backup of the docker volumes with Kopia safely and securely?
I tried to normally take the backup of /var/lib/docker/volumes, but obviously, Kopia is not allowed to access those directories due to permission issues.
Changing/tinkering with the permissions is overkill in my opinion.
Help please?

**Comment by basldfalksjdf** [comment]
As far as I know, it is not possible to somehow make Kopia get around lack-of-permissions in Linux. That is the whole point of permissions -- a user or program without appropriate permissions cannot access the files/folders, no matter if the access is legitimate. Kopia would have to use some sort of exploit to get around permissions, which is not going to happen.

Kopia needs to be able to read files in order to back them up. You can either change the permissions or download/install Kopia with an account that has permissions to access docker volumes. Alternatively, you could run Kopia within your docker volumes and backup the content of the volumes. In this case, though, you wouldn't be backing up the volumes themselves but rather the content of the volumes.

**Comment by akash07k** [comment]
Yes, so, for running Kopia with the docker container, will I have to modify the container image?

**Comment by CompuRoot** [comment]
Do not get me wrong, but I'm pretty sure this isn't an issue but a subject for discussion on kopia's forum

**Comment by akash07k** [comment]
Yes, you are right, I'll create an account on forums and will post it
